WordPress/src/main/java/org/wordpress/android/ui/activitylog/detail/ActivityLogDetailFragment.kt

```diff
@@ -31,6 +30,7 @@ class ActivityLogDetailFragment : Fragment() {
     @Inject lateinit var imageManager: ImageManager
     @Inject lateinit var notificationsUtilsWrapper: NotificationsUtilsWrapper
     @Inject lateinit var formattableContentClickHandler: FormattableContentClickHandler
+    @Inject lateinit var uiHelpers: UiHelpers
```

Comment:
How is the injected `UiHelpers` class working out? Did we end up mocking this in tests? I did a quick search and couldn't find one. Oguz mentioned in #8889 that we probably won't end up mocking this class anyway. 

I don't have a lot of context on this. I just wanted to share that I feel like this just increases verbosity and development time for a case (mocking) that might never happen.

Another concern with injected classes is they add more generated code. That adds to the 64K method limit. The more we have injected classes, the closer we get to that. Though I'm not sure if this matters in our app. 😬

Reply:
>How is the injected UiHelpers class working out? Did we end up mocking this in tests? I did a quick search and couldn't find one. Oguz mentioned in #8889 that we probably won't end up mocking this class anyway.
>I don't have a lot of context on this. I just wanted to share that I feel like this just increases verbosity and development time for a case (mocking) that might never happen.

I agree this specific utility class doesn't need to be mocked as it is (and will always be 🤞) used just in the view layer. I felt like Oguz and Mario have already considered the pros and cons so I haven't even think about converting it into static/object. For me the main benefit is that we all will get use to preferring injectable classes over static methods (which is really important for all but view-layer-only classes). But if you think we should change it, I'm not against it.

>Another concern with injected classes is they add more generated code. That adds to the 64K method limit. The more we have injected classes, the closer we get to that. Though I'm not sure if this matters in our app. 😬

I believe the multiDex is already enabled in WPAndroid.

Reply:
>which is really important for all but view-layer-only classes

This statement is too general. There are some exception in which it also makes sense to use static methods as we actually don't want to mock everything.

Reply:
> I believe the multiDex is already enabled in WPAndroid.

Cool. Didn't know that. 

> which is really important for all but view-layer-only classes
> This statement is too general. There are some exception in which it also makes sense to use static methods as we actually don't want to mock everything.

Yeah, I think it really depends. Some examples that I can think of: 

- Utility methods can most of the time be just Kotlin extensions 
- Utility methods that use Android-only APIs (e.g. File operations), we probably want these to be in classes, injectable, and mockable. These also tend to be used in ViewModels.

Reply:
This one is something I can see it go either way, so I think we just went with the safer approach. I am definitely not against making the `UiHelpers` a set of static methods instead. Since I don't feel strongly about it either way, if anyone else does, I am happy to follow their preference.

P.S: We did talk about this internally though and no strong opinions were raised iirc. I'll link that discussion to you as well.